httplib.h

```diff
@@ -9623,12 +9629,20 @@ inline bool SSLClient::initialize_ssl(Socket &socket, Error &error) {
         }
 
         if (server_certificate_verification_) {
-          if (server_certificate_verifier_) {
-            if (!server_certificate_verifier_(ssl2)) {
-              error = Error::SSLServerVerification;
-              return false;
-            }
-          } else {
+          // set default status to CheckAgain
+          SSLVerifierResponse verificationStatus = SSLVerifierResponse::CheckAgain;
+
+          if (server_certificate_verifier_) 
+            verificationStatus = server_certificate_verifier_(ssl2);
```

Comment:
This code base uses curlies even around single statements. Also, please run `clang-format`. This PR would fail #2062.